recipes/coverage-lcov/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "coverage-lcov" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3de40d620cb13e43c746126be3154252d1296787a9f2a54b14683763cf9c0c46
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - coverage-lcov = coverage_lcov.cli:main
+    - test = pytest:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6.3
+    - poetry-core >=1.0.0
+  run:
+    - python >=3.6.3
+    - toml >=0.5
+    - click >=7.1.2
+    - coverage >=5.5
+
+test:
```

Comment:
@BastianZim thank you for your help. I've restored `pip check` now.

I was wondering, is there and advantage to do the recipe directly from Github instead of from the pip package? Like, resilience in the case that the developer stopped publishing there or something of the sort?

I usually do my recipes straight from the source, but in this case I has having some issues to even build it locally so I switched to pip. Is it worth it to try to go back?

Reply:
Conda-forge actually prefers PyPI as the packages are generally slightly smaller there since only the necessary files and folders are included. That generally saves a bit of CI time which is quite valuable to us since conda-forge only has access to the open source plans. 

Resilience is a question but I've found that most packages are actually more consistently pushed to PyPI than to GitHub releases.